runtime/src/iree/hal/utils/libmpi_test.cc

```diff
@@ -88,4 +89,188 @@ TEST_F(LibmpiTest, MPIErrorToIREEStatus) {
               StatusIs(StatusCode::kInternal));
 }
 
-}  // namespace
+TEST_F(LibmpiTest, Bcast) {
+  const int N = 100;
+  char array[N];
+  int root = 1;
+
+  ASSERT_GT(world_size, 1)
+      << "This test requires mpirun with more than one node";
+
+  if (rank == root) memset(array, 'r', N);
+
+  IREE_EXPECT_OK(
+      MPI_RESULT_TO_STATUS(&symbols,
+                           MPI_Bcast(array, N, IREE_MPI_BYTE(&symbols), root,
+                                     IREE_MPI_COMM_WORLD(&symbols)),
+                           "MPI_Bcast"));
+
+  for (int i = 0; i < N; i++)
+    EXPECT_EQ(array[i], 'r') << "[" << rank << "] failed at index " << i;
+}
```

Comment:
style: braces around loops
```suggestion
  for (int i = 0; i < N; i++) {
    EXPECT_EQ(array[i], 'r') << "[" << rank << "] failed at index " << i;
  }
}
```